lib/erb_lint/linters/rubocop.rb

```diff
@@ -0,0 +1,77 @@
+# frozen_string_literal: true
+
+require 'better_html'
+require 'rubocop'
+require 'tempfile'
+
+module ERBLint
+  module Linters
+    # Run selected rubocop cops on Ruby code
+    class Rubocop < Linter
+      include LinterRegistry
+
+      BLOCK_EXPR = /\s*((\s+|\))do|\{)(\s*\|[^|]*\|)?\s*\Z/
```

Comment:
Can you leave some documentation about exactly what block expressions are captured with this regex?